CONTRIBUTING.md

```diff
@@ -13,6 +9,7 @@
 - Install the latest version of git (https://git-scm.com/downloads)
 - Install [PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell), version 6 or higher, if you plan to make public API changes or are working with generated code snippets.
 - Install [NodeJS](https://nodejs.org/) (16.x.x) if you plan to use [C# code generation](https://github.com/Azure/autorest.csharp).
+- [Fork the repository](https://docs.github.com/get-started/quickstart/fork-a-repo); work will be done in your fork and a pull request should be opened against the `main` branch of the Azure SDK for .NET repository when ready for review.
```

Comment:
There's a section on forking the repo in the management-plane specific instructions: https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#standard-process

Does this conflict with that?